docs/src/Actions/Start.md

```diff
@@ -0,0 +1,78 @@
+# Redirect User to Frequency Access
+
+To start the authentication loop, your application will generate a redirect URL for each user.
+
+## Quick Reference
+
+- Staging-Testnet: `https://testnet.frequencyaccess.com/siwa/login`
+- Production-Mainnet: `https://www.frequencyaccess.com/siwa/login`
+- Generate Signed Request Payload: [`TODO`](#todo)
+- Request Structure: [`SiwaRequest`](../DataStructures/All.md#request)
+- Signed Request Structure: [`SiwaSignedRequest`](../DataStructures/All.md#signed-request)
+
+## Step 1: Generate the Signed Request Payload
+
+You *can* generate more than one of these for use simultaneously.
+Most applications should generate and keep their needed value(s) static.
+The generated signed request payload is secured by the signature included in it, and it is not sensitive data.
+
+### Signed Request Payload Contents
+
+- A signature from one of the [Control Keys](https://docs.frequency.xyz/Identity/ControlKeys.html) of the Frequency Provider
+- The domain and path the callback will use
+- The requested permissions
+- The requested credentials
+
+### Signed Request Payload Generator
+
+TODO: Link or embed this here
+
+### Signed Request Payload Detailed Documentation
+
+For details on the Payload and how it is formed, see the [Reference Documentation](../SignatureGeneration.md).
+
+## Step 2: Build the Redirect URL Parameters
+
+### Parameter: `signedRequest`
```

Comment:
Just want to be clear, this will bea base64url encoded query parameter. is that accurate?